### ISSUE TITLE
Change to 60FPS?

### ISSUE THREAD

**Issue by nulcearc** [huytd/agar.io-clone#397]
Why not change networkUpdateFactor in config.json from 40 to 60 to change fps to 60?

**Comment by savigun** [comment]
That's a good question. But still if I change it there is some jittering in the image, like it draws at the same position twice

**Comment by abalabahaha** [comment]
Network update speed isn't the only issue, other factors like rendering also take up time.

**Comment by savigun** [comment]
Maybe add some interval between updates and apply interpolation on the client side? If you have that in mind i could help.

**Comment by dehimer** [comment]
How I understood, you have no interpolation on client side? Only change position by server data?

**Comment by iSpri** [comment]
Yes, at the moment the position is only changing by server data. But setting up movement prediction on client side shouldn't be a big problem.

**Comment by dehimer** [comment]
Ok. Thank You for fasten answer. I discovering sources becouse I making similar game and noticed, that in original agar.io animation is more smoothed.

**Comment by iSpri** [comment]
You're welcome. I'm also working on a similar game, but I would suggest you not to start from this project because there are a lot of things that aren't "production ready" in it. ;)

**Comment by dehimer** [comment]
No, I found this repo only after I started) My game with star ships))

**Comment by savigun** [comment]
Can you please be more specific about what isn't production ready?

2016-08-07 16:06 GMT+03:00 Novikov Denis Yurevich <notifications@github.com>
:

**Comment by iSpri** [comment]
Socket.io isn't really appropriate for this kind of game. Websockets packets are too big. Too much nested for loops. Movement prediction is missing.  The quadtree is recreated for each player when it could only be recreated once per update.
But the project was still really helpful to me when I was only a beginner. It was a really good startpoint. :)

**Comment by iSpri** [comment]
I'm using einaros ws. This websockets library is one of the fastest that I know.
I don't think there is a maximum packet size. The smaller the better.
